packages/@okta/vuepress-site/docs/guides/sign-users-out/define-signout-callback/react-native/addbaseuri.md

```diff
@@ -0,0 +1,17 @@
+Signing out of Okta requires the app to open a browser and navigate to the [end session endpoint](https://developer.okta.com/docs/reference/api/oidc/#logout). Okta destroys the user's session and immediately redirects back to your application. To do this, you must define a callback route for the sign-out process.
```

Comment:
I see you added a note about this being browser based flow only in another file. Do we want the same here?